### DISCUSSION TITLE
Hardware flags for an old Xeon cpu

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#5604] [General]
Have a Intel(R) Xeon(R) CPU E5-2620 v2 running on top of Supermicro X9 board. The only graphics adapter is a Matrox G200 just to display the console. 

Although i have a Coral TPU and one cam the ffmpeg process consumes 60-70%of the CPU. No HW decoding flags set. 

s /dev/dri/renderD128
ls: cannot access '/dev/dri/renderD128': No such file or directory

Any hardware acceleration flags possible or i should add some AMD 5xxx adapter just to offload? Planning to add 6 more cams

**Comment by NickM-27** [comment]
If it doesn't support intel graphics then no there's nothing to use, you'll need a discrete gpu.

**Reply by sunnysingh-84** [reply]
what detector type are you using?
